GliaWidgets/Component/Bubble/BubbleView.swift

```diff
@@ -57,6 +68,21 @@ class BubbleView: UIView {
         badgeView?.newItemCount = itemCount
     }
 
+    func adjustInnerFrame(to bounds: CGRect) {
+        innerSmallerFrame = frame
+
+        if bounds.width < frame.width {
+            let difference = frame.width - bounds.width
+            innerSmallerFrame.size.width = bounds.width
+            innerSmallerFrame.origin.x += difference / 2
+        }
+        if bounds.height < frame.height {
+            let difference = frame.height - bounds.height
+            innerSmallerFrame.size.height = bounds.height
+            innerSmallerFrame.origin.y += difference / 2
+        }
```

Comment:
If the function receives `bounds` bigger than frame the function does nothing (only frame position will be changed as a side effect). Is it expected behavior?

Reply:
Yes, this is expected. If the `bounds` can fully fit the regular `frame`, then we only use the `innerSmallerFrame` as a means to move the `frame` itself. Therefore the fact that `frame` moves anyway is not a side effect but the intention :)